test/fixtures/output/network/dns/forwarder/create-help.golden

```diff
@@ -6,15 +6,15 @@ Usage:
 Examples:
 Create a DNS forwarder.
 
-  $ confluent network dns forwarder create --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --config ForwardViaIp --domains abc.com,def.com 
+  $ confluent network dns forwarder create --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --config forward-via-ip --domains abc.com,def.com 
 
 Create a named DNS forwarder.
 
-  $ confluent network dns forwarder create my-dns-forwarder --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --config ForwardViaIp --domains abc.com,def.com 
+  $ confluent network dns forwarder create my-dns-forwarder --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --config forward-via-ip --domains abc.com,def.com 
 
 Flags:
       --gateway string           REQUIRED: Gateway ID.
-      --config string            REQUIRED: Specify the DNS forwarder configuration "ForwardViaIp".
+      --config string            REQUIRED: Specify the DNS forwarder configuration as "forward-via-ip".
```

Comment:
Seems weird to require this when there's only one option. Are you planning on adding other options soon?

Reply:
Yes we plan on supporting another option `forward-via-gcp-dns-peering`

Reply:
What about `--fowarding-type [ip|gcp-dns-peering]`

Reply:
`config` is what we have in the [API spec](https://docs.confluent.io/cloud/current/api.html#tag/DNS-Forwarders-(networkingv1)/operation/createNetworkingV1DnsForwarder), @kabraham13 thoughts on naming for this flag?

Reply:
Can we just simply abstract this away from customers? Aka not have a flag at all. The selection here is just dependent on the cloud which is a decision a customer would already have made (with the gateway) so i would ideally like to reduce the cognitive load on them needing to think about this at all

Reply:
Yes we can abstract it away since there’s only one option at the moment, but the gateway APIs (and model) haven’t been finalized so we don’t have access to cloud info just yet. I can hardcode the config for now and revisit once gateway is ready (customer experience will be the same)

Reply:
We still kind of do don't we? Its just a bit roundabout since it would be `dnsf->gateway->CCN->cloud`? 

I am also okay with what you said since there is only one option today. By the time GCP support comes (and thus we need to differentiate I imagine Gateways will be more fleshed out